app/views/users/logins/signin_save_confirmation.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="grid-row">
+  <div class="column-two-thirds">
+    <h1 class="heading-large"><%=t '.heading' %></h1>
+
+    <p><%=t('.expire_warning', expire_in_days: Rails.configuration.x.cases.expire_in_days) %></p>
```

Comment:
I think we will need to change this. The current text doesn't make it clear that it's 14 days from when you *started* the case.
Please add a TODO comment here to come back to this and use a helper method to say "Your case will be deleted in #{N} days," where N counts down from the case creation date.